utils/prng.go

```diff
@@ -1,73 +1,80 @@
 package utils
 
 import (
+	"crypto/rand"
 	"errors"
 	"golang.org/x/crypto/blake2b"
 )
 
 type PRNG interface {
-	GetClock() uint64
-	Seed(seed []byte)
-	GetSeed() []byte
 	Clock(sum []byte)
-	SetClock(sum []byte, n uint64) error
 }
 
-// prng is a structure storing the parameters used to securely and deterministically generate shared
+// OSrandPRNG is an empty struct used as a method receiver for crypto/rand Read
+type OSrandPRNG struct{}
+
+// Clock reads bytes from crypto/rand on sum.
+func (prng *OSrandPRNG) Clock(sum []byte) {
+	if _, err := rand.Read(sum); err != nil {
+		panic("crypto rand error")
+	}
+}
+
+// seededPRNG is a structure storing the parameters used to securely and deterministically generate shared
 // sequences of random bytes among different parties using the hash function blake2b. Backward sequence
 // security (given the digest i, compute the digest i-1) is ensured by default, however forward sequence
-// security (given the digest i, compute the digest i+1) is only ensured if the prng is keyed.
-type prng struct {
+// security (given the digest i, compute the digest i+1) is only ensured if the seededPRNG is keyed.
+type SeededPRNG struct {
```

Comment:
Since we are implementing the PRNG with Blake2, could a better name be `blake2PRNG`

Reply:
I thought it would be better to generalize the naming in case there are other seeded prngs added

Reply:
So let's make that an interface ? 

```suggestion
type SeededPRNG interface {
	PRNG
	Seed(seed []byte)
	GetSeed() []byte
	SetClock(sum []byte, n uint64) error
}
```

Reply:
Depending on the discussion of [this comment](https://github.com/ldsec/lattigo/pull/47#discussion_r430545437), we can change the interface name to `KeyedPRNG`. 

The `sum` variable is also not documented. Can we also simplify here ?